package-lock.json

```diff
@@ -1204,7 +1204,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
Apologies, looks like the recent version of `npm` has decided a bit of churn is required as part of the upgrade using the `install` command.